keps/sig-instrumentation/4785-crdmetrics-controller/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: CRDMetrics Controller
```

Comment:
If this is a controller, what does the `.spec` define?

Reply:
The metrics generation configuration: https://github.com/rexagod/crdmetrics/blob/main/manifests/custom-resource.yaml#L20.

Reply:
OK, but if I had to document what metrics generation configurations are, I couldn't. Ideally the KEP explains it enough that two people could write code and then docs, and produce roughly equivalent outcomes.

Reply:
Ah, so I reckon doing a rundown of the fields that the current `.spec` entails?

Reply:
Ah, so I reckon doing a rundown of the fields that the current `.spec` entails?